core/invoker/src/main/resources/application.conf

```diff
@@ -154,7 +154,7 @@ whisk {
       # The "unusedTimeout" in the ContainerProxy,
       #aka 'How long should a container sit idle until we kill it?'
       idle-container = 10 minutes
-      pause-grace = 50 milliseconds
+      pause-grace = 10 seconds
```

Comment:
This will affect the old scheduler too.
It looks fine to me as I think each downstream would use its own timeout.
But worth listening to others' opinions.

Reply:
I'm personally pro larger pause grace anyways as the default, but maybe needs to be communicated

Reply:
> This will affect the old scheduler too. It looks fine to me as I think each downstream would use its own timeout. But worth listening to others' opinions.

Yes I realize later that I have changed application.conf as well, supposed to only change `ansible/README.md`